npm/npm.go

```diff
@@ -58,6 +58,7 @@ type NetworkPolicyManager struct {
 // NewNetworkPolicyManager creates a NetworkPolicyManager
 func NewNetworkPolicyManager(config npmconfig.Config,
 	informerFactory informers.SharedInformerFactory,
+	PodFactory informers.SharedInformerFactory,
```

Comment:
nit: golang convention is to lowercase parameters. In general, lowercase means unexported/private to the package, and uppercase means the symbol is exported/accessible to other pkgs

Reply:
Thanks for pointing that out! Fixed it